doc/internals/index.rst

```diff
@@ -0,0 +1,17 @@
+================
+Sphinx internals
+================
+
+This guide contains information about the Sphinx open source project itself.
+This is where you can find information about how Sphinx is managed and learn
+how to contribute to the project.
+
+.. toctree::
+   :maxdepth: 2
+
+   contributing
+   code-of-conduct
+   release-process
+   organization
+   authors
+   changes
```

Comment:
I don't think changes is not a part of "internal". Why do you move this here?

Reply:
Fair point. Moved back to root.